src/Compiler/Service/ServiceParsedInputOps.fs

```diff
@@ -64,6 +64,9 @@ type PatternContext =
     /// Completing union case field identifier in a pattern (e.g. fun (Case (field1 = a; fie| )) -> )
     | UnionCaseFieldIdentifier of referencedFields: string list * caseIdRange: range
 
+    /// Completing a record field identifier in a pattern (e.g. fun { Field1 = a; Fie| } -> )
+    | RecordFieldIdentifier of referencedFields: (string * range) list
```

Comment:
Wanted to use `Ident` instead of the tuple, but then the context types need to be `NoComparison`, `NoEquality`.